submissions/jeff-gurpreet/scripts/controllers/aboutController.js

```diff
@@ -0,0 +1,15 @@
+(function(module) {
+  var aboutController = {};
+
+  aboutController.index = function() {
+    $('#about').show().siblings().hide();
+
+    // TODO: Call the function that loads (or 'requests') our repo data.
+    //  Pass in some view function as a higher order callback, so our repos
+    //  will render after the data is loaded.
+    repos.requestRepos(repoView.index());
```

Comment:
In other words, here you're immediately calling repoView.index (because of the ()), running that function, then passing the results to repos.requestRepos. That's probably not what you want to do.